compiler/luci/import/src/Importer.test.cpp

```diff
@@ -309,3 +309,44 @@ TEST(TensorFlowLiteImport, corrupted_plan_NEG)
 
   ASSERT_ANY_THROW(import.importModule(model_ptr));
 }
+
+/**
+ * This test checks that empty execution plan entry induce exception
+ */
+TEST(TensorFlowLiteImport, corrupted_plan_entry_NEG)
```

Comment:
Q) how does luci import tflite? I didn't notice this before... _luci_ is all about circle, not tflite.

Reply:
Wow

I copied this name from old test `TEST(TensorFlowLiteImport, Dummy)` and didn't pay much attention to the name.
All tests in this file are in `TensorFlowLiteImport` suite, but actually operate on buffers create with `circle` schema.

May I rename them in next PR?

Reply:
> May I rename them in next PR?

Yes, certainly :)